test/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration.Tests/ExtensionsTests/StringExtensionTest.cs

```diff
@@ -45,60 +45,94 @@ public static IEnumerable<object[]> GetTestCasesForToCamelCaseShouldSucceed()
             };
         }
 
-        public static IEnumerable<object[]> GetTestCasesForRemoveRoslynDuplicateStringShouldSucceed()
+        public static IEnumerable<object[]> GetTestCasesForRemoveRoslynDuplicateStringShouldUpdateCorrectly()
         {
+            yield return new object[]
+            {
+                "String with even length starting from -",
+                "abc-ef-efg",
+                "abc-ef-efg"
+            };
+
+            yield return new object[]
+            {
+                "String with odd length starting from -",
```

Comment:
Can we say "string with the special-character-containing part having even length starting with -"?